spinta/types/type.py

```diff
@@ -235,6 +241,33 @@ def load_type(context: Context, prop: Node, data: dict, manifest: Manifest):
     return load(context, type, data, manifest)
 
 
+@load.register()
+def load(context: Context, type: Type, value: object) -> object:
+    # loads value to python native value according to given type
+    return type.load(value)
+
+
+@load.register()
+def load(context: Context, type: Array, value: object) -> object:
+    # loads value into native python list, including all list items
+    array_item_type = type.items.type
+    loaded_array = type.load(value)
+    new_loaded_array = []
+    for item in loaded_array:
+        new_loaded_array.append(load(context, array_item_type, item))
+    return new_loaded_array
+
+
+@load.register()
+def load(context: Context, type: Object, value: object) -> object:
+    # loads value into native python dict, including all dict's items
+    loaded_obj = type.load(value)
+    new_loaded_obj = {}
+    for k, v in type.properties.items():
+        new_loaded_obj[k] = load(context, v.type, loaded_obj[k])
```

Comment:
Keep in mind, that here value is at the very early stage and `k` might not be present in `loaded_obj`, but because of that loading should not raise an error. You should only load given items and make sure, that there are no unknown (not in `type.properties`) items.